tutorials/ardu-pi-serial-part-1/index.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Locally Connected Microcontrollers and Real-Time Analytics (Part 1 of 2)
```

Comment:
For the 3 images in this file, you need to prefix with `https://storage.googleapis.com/gcp-community/tutorials/ardu-pi-serial-part-1/`

e.g. https://storage.googleapis.com/gcp-community/tutorials/ardu-pi-serial-part-1/ardu-pi.png

Reply:
Done